packages/react-native-editor/__device-tests__/helpers/utils.js

```diff
@@ -419,14 +419,42 @@ const tapStatusBariOS = async ( driver ) => {
 };
 
 const selectTextFromElement = async ( driver, element ) => {
-	await longPressMiddleOfElement( driver, element );
+	const timeout = 1000;
+
+	// On Android we can't "locate" the context menu options,
+	// To avoid having fixed coordinates to be able to
+	// select all text, it just selects all text by
+	// long-pressing and dragging.
+	if ( isAndroid() ) {
+		await element.click();
+
+		const location = await element.getLocation();
+		const size = await element.getSize();
+		const paddingPercentage = 0.2;
+		const leftPaddingOffset = size.width * paddingPercentage;
+		const startX = location.x + leftPaddingOffset;
+		const endX = location.x + size.width;
+		const centerY = location.y + size.height / 2;
+
+		await driver
+			.action( 'pointer', {
+				parameters: { pointerType: 'touch' },
+			} )
+			.move( { x: startX, y: centerY } )
+			.down()
+			.pause( timeout )
+			.move( { x: endX, y: centerY, duration: timeout } )
+			.up()
+			.pause( timeout )
```

Comment:
What is the significant of the two `pause` commands in this sequence? Why is the `timeout` value used for `duration`?

Reply:
> First long-press at the start of the element (that's the first pause).

I followed that, but I didn't understand why it was necessary. I played with selecting text on Android just now and see why it is required — if you do not pause, Android does not begin text selection, but instead relocates the cursor. 

> The timeout for the duration should really have a different naming but this value is needed so it doesn't move too quickly so it highlights all of the text to the end. Since it was using the same 1000 value I reused this variable but we can have a different one or just the value directly.

I suggest inlining a `1000` value. Reusing `timeout` conveys it has some noteworthy relationship to the parameter or `pause` commands. 

> The last pause is to wait for the context menu to show up. Some tests rely on checking the caret selection color.

Ah, that makes sense.